SMLHelper/Handlers/CustomFishHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using System;
+    using System.Collections.Generic;
+    using SMLHelper.V2.Assets;
+    using System.IO;
+
+    /// <summary>
+    /// Class to manage registering of fish into the game
+    /// </summary>
+    public static class CustomFishHandler
+    {
+        /// <summary>
+        /// A list of all the custom fish that have so far been registered into the game. This includes ones from mods that may have been loaded earlier.
+        /// It is mainly used by CustomFishPatcher to spawn fish in
+        /// </summary>
+        public static List<TechType> fishTechTypes = new List<TechType>();
+
+        /// <summary>
+        /// Registers a CustomFish object into the game
+        /// </summary>
+        /// <param name="fish">The CustomFish that you are registering</param>
+        /// <returns>The TechType created using the info from your CustomFish object</returns>
+        public static TechType RegisterFish(CustomFish fish)
```

Comment:
Would this work with `CustomFishPrefab`?

Reply:
This method creates a `CustomFishPrefab` which is registered using `PrefabHandler`. If the user created their own `CustomFishPrefab` instead of using `CustomFish`, they will effectively need to do everything in this method manually

Reply:
Ah, I see. Thanks for the clarification